includes/definitions/discovery/apc.yaml

```diff
@@ -84,6 +84,38 @@ modules:
                     states:
                         - { descr: open, graph: 1, value: 1, generic: 0 }
                         - { descr: closed, graph: 1, value: 2, generic: 2 }
+                -
+                    oid: rPDU2PhaseStatusTable
+                    value: rPDU2PhaseStatusLoadState
+                    num_oid: .1.3.6.1.4.1.318.1.1.26.6.3.1.4.
+                    index: 'rPDU2PhaseStatusLoadState.{{ $index }}'
+                    descr: 'Load state #{{ $index }}'
+                    states:
+                        - { descr: lowLoad, graph: 1, value: 1, generic: 1 }
+                        - { descr: normal, graph: 1, value: 2, generic: 0 }
+                        - { descr: nearOverload, graph: 1, value: 3, generic: 1 }
+                        - { descr: overload, graph: 1, value: 4, generic: 2 }
+                -
+                    oid: rPDU2DeviceStatusTable
+                    value: rPDU2DeviceStatusLoadState
+                    num_oid: .1.3.6.1.4.1.318.1.1.26.4.3.1.4.
+                    index: 'rPDU2DeviceStatusLoadState.{{ $index }}'
+                    descr: rPDU2DeviceStatusName
+                    states:
+                        - { descr: lowLoad, graph: 1, value: 1, generic: 1 }
+                        - { descr: normal, graph: 1, value: 2, generic: 0 }
+                        - { descr: nearOverload, graph: 1, value: 3, generic: 1 }
+                        - { descr: overload, graph: 1, value: 4, generic: 2 }
+                        - { descr: notsupported, graph: 1, value: 5, generic: 3 }
+                -
+                    oid: rPDU2DeviceStatusTable
+                    value: rPDU2DeviceStatusPowerSupplyAlarm
+                    num_oid: .1.3.6.1.4.1.318.1.1.26.4.3.1.12.
+                    index: 'rPDU2DeviceStatusPowerSupplyAlarm.{{ $index }}'
+                    descr: rPDU2DeviceStatusName
```

Comment:
Perhaps descr should be "Power Supply State", rather than the sysName of the device.